source/transfer_list.rst

```diff
@@ -902,5 +902,42 @@ This manifest contains the SPMC attribute node consumed by the SPMD
      - hdr_size
      - Holds a SPMC manifest image in DT format.
 
+
+**Executable entry point information entry layout (XFERLIST_EXEC_EP_INFO)**
+
+This entry type holds an array of `entry_point_info_t` structures.
+`entry_point_info_t` is a data structure containing information used by a
+bootloader to initialize the execution state of an image. This entry provides
+the consumer with all information to execute subsequent images.
+
+.. _tab_entry_point_info:
+.. list-table:: Entry point info type layout
+   :widths: 2 2 2 8
+
+   * - Field
+     - Size (bytes)
+     - Offset (bytes)
+     - Description
+
+   * - tag_id
+     - 0x3
+     - 0x0
+     - The tag_id field must be set to **0x102**.
+
+   * - hdr_size
+     - 0x1
+     - 0x3
+     - |hdr_size_desc|
+
+   * - data_size
```

Comment:
Is this needed? It looks like the value is always 8 so it can be omitted.

Reply:
This is a variable length inline array, not a pointer.